Formula/ocaml-findlib.rb

```diff
@@ -0,0 +1,26 @@
+class OcamlFindlib < Formula
+  desc "OCaml library manager"
+  homepage "http://projects.camlcity.org/projects/findlib.html"
+  url "http://download.camlcity.org/download/findlib-1.7.3.tar.gz"
+  sha256 "d196608fa23c36c2aace27d5ef124a815132a5fcea668d41fa7d6c1ca246bd8b"
+
+  depends_on "ocaml"
```

Comment:
Is this actually a run-time requirement or is it suitable for use with other OCaml besides brew's?

Reply:
- `./configure` fails if no OCaml installation is in the `PATH`
- The Findlib binaries get the path to the `lib` directory of the OCaml installation detected by `./configure` hardcoded into it.
- `META` files are generated for consumption at run time based on the capabilities of this OCaml installation.

So, while it seems to be possible to override all of these defaults, it is clear that a given Findlib installation is meant to be used in conjunction with its associated OCaml installation.

Reply:
OK.